src/App.js

```diff
@@ -1,29 +1,22 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+/* eslint-disable */
+import Calculator from './components/Calculator';
+/* eslint-enable */
```

Comment:
- [ ] Please don't disable eslint because it prevents your from producing those bugs which can break your application :thinking: